libs/rapidsandrivers-extras/src/main/kotlin/rapidsandrivers/migrering/MigreringRequest.kt

```diff
@@ -0,0 +1,17 @@
+package rapidsandrivers.migrering
+
+import no.nav.etterlatte.libs.common.behandling.Persongalleri
+import no.nav.etterlatte.libs.common.person.Folkeregisteridentifikator
+import java.time.LocalDateTime
+
+data class MigreringRequest(
+    val pesysId: PesysId,
+    val enhet: Enhet,
+    val fnr: Folkeregisteridentifikator,
+    val mottattDato: LocalDateTime,
```

Comment:
Er dette når migreringsrequesten ble mottat? Vi har mottatdato for søknader i doffen, fordi datoen nav mottar søknaden kan påvirke når ytelsen skal gjelde fra, men for migrering så det vel andre datoer som styrer når det skal gjelde fra.

Reply:
Godt spørsmål. Har tenkt på det som mottattdatoen i Pesys, og at det er det som er mest interessant for oss. Men kanskje det er relevant å veta også når vi migrerte 🤔

Reply:
Tror det er dårlig kvalitet på mottat-datoen i pesys, for det er noe som ble innført i ettertid, så det er noe å ha i bakhodet :)